server/src/server.ts

```diff
@@ -21,10 +20,10 @@ const serverContext = { app, env, prisma };
 
 trpcRoutes(serverContext);
 
-app.use(express.static(path.resolve(__dirname, '../../../client/dist')));
+app.use(express.static('dist'));
```

Comment:
There were some issue with express not setting the correct header for mimeType.
js files were marked as text/html (which was blocked by the client).
In any case, since it's a monolith, it should be just as easy to create a `client-build` script that builds client and copy the server/dist.